features/support/openhab_rest.rb

```diff
@@ -107,4 +107,12 @@ def self.item_groups(body, item)
     body[:groupNames] = groups unless groups.empty?
     body[:groupType] = item.group_type unless item.group_type.to_s.empty?
   end
+
+  def self.link_item(item_name:, channel_uid:)
+    body = { itemName: item_name, channelUID: channel_uid }
+    escaped_channel_uid = URI.encode_www_form_component(channel_uid)
+    put("/rest/links/#{item_name}/#{escaped_channel_uid}", headers: json, body: body.to_json)
+    # For some reason linking via the api doesn't create the metadata, this ensures it exists
+    put("/rest/items/#{item_name}/metadata/channel", headers: json, body: { value: channel_uid }.to_json)
```

Comment:
Do we still need to add the metadata? I tried adding an item through the GUI and linked it to a thing, and then tried accesing the A_Test.meta['channel']&.value and it returned nil.

Reply:
Yeah, this can go. In the current implementation it doesn't matter anymore.